.nf-core.yml

```diff
@@ -23,7 +21,7 @@ template:
   outdir: .
   skip_features:
     - igenomes
-  version: 2.12.0
+  version: 2.13.0dev
```

Comment:
Should that be `2.13.0`, or is that later?

Reply:
That seems to have slipped the version bump indeed! I'll correct that.